js-webapp/README.md

```diff
@@ -90,17 +86,13 @@ which takes a ``double``, and returns another ``double``. Furthermore, ``NewtonR
 #ifndef H_NEWTONRAPHSON_H
```

Comment:
```suggestion
#ifndef H_NEWTONRAPHSON_HPP
```

Reply:
Can H_ be removed? Seems duplication to me
```suggestion
#ifndef NEWTONRAPHSON_HPP
```

Reply:
will do later